### ISSUE TITLE
Client logs fail WCAG AA compliance due to bad contrast ratio.

### ISSUE THREAD

**Issue by MiniOranges** [MiniOranges/starwatch#43]
See: 
![image](https://user-images.githubusercontent.com/56451580/121562683-4e17ba80-c9df-11eb-9e7a-e9015e979fd0.png)

The color is ``rgb(85,85,85)`` on a background of ``rgb(43,43,43)``
